mlflow/pyfunc/backend.py

```diff
@@ -282,12 +289,79 @@ def can_score_model(self):
             # Can not find conda
             return False
 
+    def generate_dockerfile(
+        self,
+        model_uri,
+        output_path="mlflow-dockerfile",
+        install_mlflow=False,
+        mlflow_home=None,
+        enable_mlserver=False,
+    ):
+        copy_model_into_container = self.copy_model_into_container_wrapper(
+            model_uri, install_mlflow, enable_mlserver
+        )
+        pyfunc_entrypoint = _pyfunc_entrypoint(
+            self._env_manager, model_uri, install_mlflow, enable_mlserver
+        )
+
+        mlflow_home = os.path.abspath(mlflow_home) if mlflow_home else None
+
+        is_conda = self._env_manager == _EnvManager.CONDA
+        setup_miniconda = ""
+        setup_pyenv_and_virtualenv = ""
+
+        if is_conda:
+            setup_miniconda = SETUP_MINICONDA
+        else:
+            setup_pyenv_and_virtualenv = SETUP_PYENV_AND_VIRTUALENV
+
+        os.makedirs(output_path, exist_ok=True)
+
+        _logger.debug("Created all folders in path", extra={"output_directory": output_path})
+        install_mlflow = _get_mlflow_install_step(output_path, mlflow_home)
+
+        custom_setup_steps = ()
+        if model_uri:
+            custom_setup_steps = copy_model_into_container(output_path)
+        else:
+            custom_setup_steps = ""
```

Comment:
```suggestion
        custom_setup_steps = copy_model_into_container(output_path)
```

Can we remove `if-else`?